index.js

```diff
@@ -59,10 +58,6 @@ module.exports = ({
     })
   }
 
-  const exPool = new ExchangePoolServer({
-    port: exPoolServerPort
-  })
-
   const api = new APIWSServer({
     db: apiDB,
     port: wsServerPort,
```

Comment:
should we remove also this one?
```javascript
 exPoolURL: `http://localhost:${exPoolServerPort}`,
```

Reply:
ok exPoollURL needs to be removed here and in [APIWSServer constructor](https://github.com/bitfinexcom/bfx-hf-server/pull/54/files#diff-e65f271b76ad71497d8d1e9cd921591592a766057407807e3ba61e8cf7b1c097R38)